src/components/Sections/GroupedSectionsView.tsx

```diff
@@ -34,6 +36,10 @@ export const GroupedSectionView = ({
 
       <div className="shortcuts-list u-w-100 u-mv-3 u-mv-2-t u-mh-auto u-flex-justify-center">
         {sections.map(section => {
+          // We add a failsafe in the view to avoid rendering an empty section
```

Comment:
It would be nice to extract all this logic into its own component like `GroupedSectionTile` or something else

Reply:
updated